tests/Unit/AcceleratorViewCopy/avstress_0x18.cpp

```diff
@@ -68,25 +68,33 @@ int main(){
         }
     }
 
-    if (testsToRun & 0x8) {
+// Create a vector of hc::completion_future for async copy synconization
+    std::vector<hc::completion_future> cfs;
+    
+    if (testsToRun & 0x18) {
         for(uint32_t i=0;i<testIters;i++){
             if ((i%1000 == 0)) {
                 printf ("info: running Test4 %5d/%5d\n", i, testIters);
             }
-            Test4(av);
+            cfs.push_back(Test4(av));
         }
+        for(uint32_t i=0;i<testIters;i++)
+	    cfs[i].wait();
```

Comment:
Instead of keeping all the completion_futures (unless you want to stress the completion_future creation) in a vector, you could simply call the wait() on the accelerator_view at the end av.wait();
which will wait on the last command send to the accelerator_view